201303开始 www.szse.cn
201804开始 docs.static.szse.cn
https://akshare.akfamily.xyz/data/stock/stock.html#id9

In [ ]:
import akshare as ak

In [ ]:
ak.stock_sse_summary()

In [ ]:
df1 = ak.stock_szse_sector_summary(symbol="当年", date="202501").drop('项目名称-英文',axis=1)
df2 = ak.stock_szse_sector_summary(symbol="当年", date="202401").drop('项目名称-英文',axis=1)

In [ ]:
((((df1['成交金额-人民币元']/df1['交易天数']))/100000000)-(((df2['成交金额-人民币元']/df2['交易天数']))/100000000)).round(2)

In [ ]:
df1

In [ ]:
(((df1['成交金额-人民币元']/df1['交易天数']))/100000000).round(2)

In [ ]:
ak.stock_szse_sector_summary(symbol="当月", date="201301").drop('项目名称-英文',axis=1)

In [ ]:
ak.stock_szse_sector_summary(symbol="当月", date="201701").drop('项目名称-英文',axis=1)

In [ ]:
import pandas as pd
import requests
from bs4 import BeautifulSoup
from io import BytesIO, StringIO

初始化需要修改代码，2018年4月后数据不用修改

In [ ]:
def stock_szse_sector_summary(
    symbol: str = "当月", date: str = "202501"
) -> pd.DataFrame:
    """
    深圳证券交易所-统计资料-股票行业成交数据
    https://docs.static.szse.cn/www/market/periodical/month/W020220511355248518608.html
    :param symbol: choice of {"当月", "当年"}
    :type symbol: str
    :param date: 交易年月
    :type date: str
    :return: 股票行业成交数据
    :rtype: pandas.DataFrame
    """
    url = "https://www.szse.cn/market/periodical/month/index.html"
    r = requests.get(url)
    r.encoding = "utf8"
    soup = BeautifulSoup(r.text, features="lxml")
    tags_list = soup.find_all(name="div", attrs={"class": "g-container"})[1].find_all(
        "script"
    )
    tags_dict = [
        eval(
            item.string[item.string.find("{") : item.string.find("}") + 1]
            .replace("\n", "")
            .replace(" ", "")
            .replace("value", "'value'")
            .replace("text", "'text'")
        )
        for item in tags_list
    ]
    date_url_dict = dict(
        zip(
            [item["text"] for item in tags_dict],
            [item["value"][2:] for item in tags_dict],
        )
    )
    date_format = "-".join([date[:4], date[4:]])
    url = f"https://www.szse.cn/market/periodical/month/{date_url_dict[date_format]}"
    r = requests.get(url)
    r.encoding = "utf8"
    soup = BeautifulSoup(r.text, features="lxml")
    url = [
        item for item in soup.find_all("a") if item.get_text() == "股票行业成交数据"
    ][0]["href"]

    if 'http' in url :
        pass
    else:
        url = 'https://www.szse.cn'+url

    if symbol == "当月":
        r = requests.get(url)
        r.encoding = 'GBK'
        temp_df = pd.read_html(StringIO(r.text), encoding="gbk")[0]
        temp_df.columns = [
            "项目名称",
            "项目名称-英文",
            "交易天数",
            "成交金额-人民币元",
            "成交金额-占总计",
            "成交股数-股数",
            "成交股数-占总计",
            "成交笔数-笔",
            "成交笔数-占总计",
        ]
    else:
        temp_df = pd.read_html(url, encoding="gbk")[1]
        temp_df.columns = [
            "项目名称",
            "项目名称-英文",
            "交易天数",
            "成交金额-人民币元",
            "成交金额-占总计",
            "成交股数-股数",
            "成交股数-占总计",
            "成交笔数-笔",
            "成交笔数-占总计",
        ]

    temp_df["交易天数"] = pd.to_numeric(temp_df["交易天数"], errors="coerce")
    temp_df["成交金额-人民币元"] = pd.to_numeric(
        temp_df["成交金额-人民币元"], errors="coerce"
    )
    temp_df["成交金额-占总计"] = pd.to_numeric(
        temp_df["成交金额-占总计"], errors="coerce"
    )
    temp_df["成交股数-股数"] = pd.to_numeric(temp_df["成交股数-股数"], errors="coerce")
    temp_df["成交股数-占总计"] = pd.to_numeric(
        temp_df["成交股数-占总计"], errors="coerce"
    )
    temp_df["成交笔数-笔"] = pd.to_numeric(temp_df["成交笔数-笔"], errors="coerce")
    temp_df["成交笔数-占总计"] = pd.to_numeric(
        temp_df["成交笔数-占总计"], errors="coerce"
    )
    return temp_df

In [ ]:
stock_szse_sector_summary(symbol="当月", date="201501")

In [ ]:
gen = (f"{year}{month:02d}" for year in range(2000,2101) for month in range(1,13))

In [ ]:
# 生成2010-2015年所有月份的序列 
ylist = [f"{year}{month:02d}" for year in range(2013, 2025) for month in range(1,13)] + ['202501']

In [ ]:
ddf = []

In [ ]:
for n in ylist:
    df = ak.stock_szse_sector_summary(symbol="当月", date= n ).drop('项目名称-英文',axis=1)
    df['日期'] = n
    ddf.append(df)



In [ ]:
dff = pd.concat(ddf)

In [ ]:
dff.set_index('日期')

In [ ]:
from sqlalchemy import create_engine
eng = create_engine('postgresql+psycopg://sa:11111111@10.3.18.56:5432/StockBas')

In [ ]:
dff.set_index('日期').to_sql('szSum', eng, if_exists="replace")

In [ ]:
ddf = pd.DataFrame(columns=[ '日期','项目名称', '交易天数', '成交金额-人民币元', '成交金额-占总计', '成交股数-股数', '成交股数-占总计',
       '成交笔数-笔', '成交笔数-占总计'],dtype='object')

In [ ]:
n = ylist[0]
df = ak.stock_szse_sector_summary(symbol="当月", date= n ).drop('项目名称-英文',axis=1)
df['日期'] = n
# ddf.append(df)
ddf = pd.concat([ddf, df],axis=0,ignore_index=True,copy=False)


In [ ]:
import pandas as pd
df = pd.DataFrame()

个股信息查询

In [ ]:
ak.stock_individual_info_em(symbol="600996")

行情报价

In [ ]:
ak.stock_bid_ask_em(symbol="600996")

实时行情数据-东财 沪深京A stock_zh_a_spot_em()

沪A股  _sh_
        深A股 _sz_
        京A股 _bj_
        新股 _new_  
        创业板 _cy_  
        科创板 _kc_



In [ ]:
ak.stock_zh_a_spot_em()

 单次返回指定股票最近一个交易日的分时数据, 包含盘前数据

In [ ]:
ak.stock_intraday_em(symbol="000409")

In [ ]:
ak.stock_intraday_em(symbol="000409").groupby('买卖盘性质').sum()

目标地址: https://data.eastmoney.com/gsrl/gsdt.html
公司动态

In [ ]:
ak.stock_gsrl_gsdt_em(date="20250211")

沪深新股
目标地址: https://quote.eastmoney.com/center/gridlist.html#newshares

In [ ]:
ak.stock_zh_a_new_em()

机构调研http://data.eastmoney.com/jgdy/tj.html
http://data.eastmoney.com/jgdy/xx.html

单次返回所有历史数据

In [ ]:
ak.stock_jgdy_tj_em(date="20210128")

In [ ]:
ak.stock_jgdy_detail_em(date="20241211")

In [ ]:
ak.stock_zyjs_ths(symbol="600996")

In [ ]:
ak.stock_zygc_em(symbol="SH600996")

In [ ]:
ak.stock_zygc_ym(symbol="000001")

上市公司质押比例
目标地址: https://data.eastmoney.com/gpzy/pledgeRatio.aspx 查询具体交易日

重要股东股权质押明细 https://data.eastmoney.com/gpzy/pledgeDetail.aspx 


In [ ]:
ak.stock_gpzy_pledge_ratio_em(date="20250207")

In [ ]:
ak.stock_gpzy_pledge_ratio_detail_em()

In [ ]:
def calculate_alert_liquidation_price(initial_price, pledge_rate, interest_rate, alert_ratio=1.6, liquidation_ratio=1.4):
    """
    initial_price: 初始股价（元）
    pledge_rate: 质押率（0-1）
    interest_rate: 融资利率（年化，0-1）
    alert_ratio: 预警线系数（默认160%）
    liquidation_ratio: 平仓线系数（默认140%）
    """
    factor = initial_price * pledge_rate * (1 + interest_rate)
    alert_price = factor * alert_ratio
    liquidation_price = factor * liquidation_ratio
    return alert_price, liquidation_price 

In [ ]:
calculate_alert_liquidation_price(8.2, 0.4999, 0.06)

个股商誉明细 目标地址: https://data.eastmoney.com/sy/list.html

行业商誉 目标地址: https://data.eastmoney.com/sy/hylist.html


In [ ]:
ak.stock_sy_em(date="20221231")


In [ ]:
ak.stock_sy_hy_em(date="20231231")

分析师指数 目标地址: https://data.eastmoney.com/invest/invest/list.html
 分析师详情 https://data.eastmoney.com/invest/invest/11000257131.html

In [ ]:
ak.stock_analyst_rank_em() 

In [ ]:
ak.stock_analyst_detail_em(analyst_id="11000280036", indicator="最新跟踪成分股")

千股千评 目标地址: https://data.eastmoney.com/stockcomment/

In [ ]:
ak.stock_comment_em()

In [ ]:
ak.stock_comment_detail_zlkp_jgcyd_em(symbol="600996")

In [ ]:
ak.stock_comment_detail_zhpj_lspf_em(symbol="600996")

In [ ]:
ak.stock_comment_detail_scrd_focus_em(symbol="600996")

In [ ]:
ak.stock_comment_detail_scrd_desire_daily_em(symbol="600996")

停复牌信息

In [ ]:
ak.stock_tfp_em()

个股新闻 目标地址: https://so.eastmoney.com/news/s
财经内容精选 目标地址: https://cxdata.caixin.com/pc/

In [ ]:
ak.stock_news_em(symbol="600996")

In [ ]:
ak.stock_news_main_cx()

年报季报 目标地址: http://data.eastmoney.com/bbsj/202412/yjbb.html

date="20200331"; choice of {"0331一季报", "0630中报", "0930三季报", "1231年报"}; 从 20100331 开始


In [ ]:
ak.stock_yjbb_em(date="20241231")

In [ ]:
ak.stock_yjkb_em(date="20241231")

In [ ]:
ak.stock_yjyg_em(date="20241231")

预约披露时间-东方财富

In [ ]:
ak.stock_yysj_em(symbol="沪深A股", date="20241231")

预约披露时间-巨潮资讯 目标地址: http://www.cninfo.com.cn/new/commonUrl?url=data/yypl

period="2021年报"; 近四期的财务报告; e.g., choice of {"2021一季", "2021半年报", "2021三季", "2021年报"}


In [ ]:
ak.stock_report_disclosure(market="沪深京", period="2024年报")

上市公司行业归属的变动情况-巨潮资讯

In [ ]:
ak.stock_industry_change_cninfo(symbol="002594", start_date="20050101", end_date="20250212")

公司股本变动-巨潮资讯

In [ ]:
fd = ak.stock_share_change_cninfo(symbol="600996", start_date="20050101", end_date="20250212")

In [ ]:
df = ak.stock_profile_cninfo(symbol="600996")

高管持股

In [ ]:
ak.stock_ggcg_em(symbol="全部")

分红配送

In [ ]:
ak.stock_fhps_em(date="20241231")

In [ ]:
ak.stock_fhps_detail_em(symbol="600409")

In [ ]:
ak.stock_fhps_detail_ths(symbol="600409")

资金流向 

symbol="即时"; choice of {“即时”, "3日排行", "5日排行", "10日排行", "20日排行"}

个股资金流：individual  概念资金流：concept 行业资金流：industry 

In [ ]:
ak.stock_fund_flow_individual(symbol="即时")

筹码分布 adjust=""; choice of {"qfq": "前复权", "hfq": "后复权", "": "不复权"}

In [ ]:
ak.stock_cyq_em(symbol="600996", adjust="")

个股研报

In [ ]:
ak.stock_research_report_em(symbol="600409")


股东户数

In [ ]:
ak.stock_zh_a_gdhs(symbol="20241231")

In [ ]:
ak.stock_zh_a_gdhs_detail_em(symbol="600996")

限售股解禁详情

In [ ]:
ak.stock_restricted_release_detail_em(start_date="20240326", end_date="20250212")

股票列表-A股 目标地址: 沪深京三个交易所

In [ ]:
ak.stock_info_a_code_name()

In [ ]:
ak.stock_info_sh_name_code(symbol="主板A股")

In [ ]:
ak.stock_info_sz_name_code(symbol="A股列表")

In [ ]:
ak.stock_info_bj_name_code()

股票更名

In [ ]:
ak.stock_info_change_name(symbol="000001")

symbol="全称变更"; choice of {"全称变更", "简称变更"}


In [ ]:
ak.stock_info_sz_change_name(symbol="全称变更")

symbol="证监会行业分类"; choice of {"证监会行业分类", "国证行业分类"}

In [ ]:
ak.stock_industry_pe_ratio_cninfo(symbol="证监会行业分类", date="20250212")

In [ ]:
ak.stock_hold_num_cninfo(date="20240930")

实际控制人持股变动

symbol="全部"; choice of {"单独控制", "实际控制人", "一致行动人", "家族控制", "全部"}; 从 2010 开始


In [ ]:
ak.stock_hold_control_cninfo(symbol="全部")

对外担保 symbol="全部"; choice of {"全部", "深市主板", "沪市", "创业板", "科创板"}



In [ ]:
ak.stock_cg_guarantee_cninfo(symbol="全部", start_date="20240601", end_date="20250212")

股权质押

In [ ]:
ak.stock_cg_equity_mortgage_cninfo(date="20250212")

A 股个股指标
symbol="000001"; 参见 ak.stock_a_indicator_lg(symbol="all") 获取股票代码

pe	市盈率
pe_ttm	市盈率TTM
pb		市净率
ps	市销率
ps_ttm	市销率TTM
dv_ratio		股息率
dv_ttm		股息率TTM
total_mv		总市值

In [ ]:
ak.stock_a_indicator_lg(symbol="600409")

A 股股息率 symbol="上证A股"; choice of {"上证A股", "深证A股", "创业板", "科创板"}


In [ ]:
ak.stock_a_gxl_lg(symbol="深证A股")

大盘拥挤度

In [ ]:
ak.stock_a_congestion_lg()

股债利差

In [ ]:
ak.stock_ebs_lg()

A 股等权重与中位数市盈率

In [ ]:
ak.stock_a_ttm_lyr()

A 股等权重与中位数市净率

In [ ]:
ak.stock_a_all_pb()

主板市盈率 symbol="上证"; choice of {"上证", "深证", "创业板", "科创版"}


In [ ]:
ak.stock_market_pe_lg(symbol="上证")

指数市盈率symbol="上证50"; choice of {"上证50", "沪深300", "上证380", "创业板50", "中证500", "上证180", "深证红利", "深证100", "中证1000", "上证红利", "中证100", "中证800"}


In [ ]:
ak.stock_index_pe_lg(symbol="上证50")

主板市净率 symbol="上证"; choice of {"上证", "深证", "创业板", "科创版"}


In [ ]:
ak.stock_market_pb_lg(symbol="上证")

In [ ]:
ak.stock_index_pb_lg(symbol="上证50")

个股估值

In [ ]:
ak.stock_value_em(symbol="300766")

创新高和新低的股票数量 symbol="all"; {"all": "全部A股", "sz50": "上证50", "hs300": "沪深300", "zz500": "中证500"}


In [ ]:
ak.stock_a_high_low_statistics(symbol="zz500")

破净股统计 symbol="全部A股"; choice of {"全部A股", "沪深300", "上证50", "中证500"}


In [ ]:
ak.stock_a_below_net_asset_statistics(symbol="全部A股")

In [ ]:
ak.stock_report_fund_hold(symbol="基金持仓", date="20241231")

机构买卖每日统计 
机构席位追踪 symbol="近一月"; choice of {"近一月", "近三月", "近六月", "近一年"}

In [ ]:
ak.stock_lhb_jgmmtj_em(start_date="20250101", end_date="20250212")

In [ ]:
ak.stock_lhb_jgstatistic_em(symbol="近一月")

活跃 A 股统计 symbol='近三月'; choice of {'近一月', '近三月', '近六月', '近一年'}

In [ ]:
ak.stock_dzjy_hygtj(symbol='近三月')

融资融券汇总

标的证券名单及保证金比例查询

In [ ]:
ak.stock_margin_ratio_pa(date="20250213")

In [ ]:
ak.stock_margin_account_info()

In [ ]:
ak.stock_margin_sse(start_date="20250212", end_date="20250212")

In [ ]:
ak.stock_margin_szse(date="20250212")

In [ ]:
ak.stock_margin_detail_sse(date="20250212")

In [ ]:
ak.stock_margin_detail_szse(date="20250212")

In [ ]:
ak.stock_margin_underlying_info_szse(date="20250212")

盈利预测-东方财富  symbol="", 默认为获取全部数据; symbol="船舶制造", 则获取具体行业板块的数据; 行业板块可以通过 ak.stock_board_industry_name_em() 接口获取


In [ ]:
ak.stock_profit_forecast_em()

indicator="预测年报每股收益"; choice of {"预测年报每股收益", "预测年报净利润", "业绩预测详表-机构", "业绩预测详表-详细指标预测"}


In [ ]:
ak.stock_profit_forecast_ths(symbol="600519", indicator="预测年报每股收益")

东方财富-概念板块

In [ ]:
ak.stock_board_concept_name_em()

东方财富-成份股 symbol="车联网"; 可以通过调用 ak.stock_board_concept_name_em() 查看东方财富-概念板块的所有行业名称


In [ ]:
ak.stock_board_concept_cons_em(symbol="高带宽内存")

东方财富-指数 symbol="长寿药"; 可以通过调用 ak.stock_board_concept_name_em() 查看东方财富-概念板块的所有概念代码
 
 period="5"; choice of {"1", "5", "15", "30", "60"}


In [ ]:
ak.stock_board_concept_hist_em(symbol="高带宽内存", period="daily", start_date="20220101", end_date="20250212", adjust="")

东方财富-行业板块

In [ ]:
ak.stock_board_industry_name_em()

东方财富-成份股

In [ ]:
ak.stock_board_industry_cons_em(symbol="电机")

In [ ]:
ak.stock_board_industry_hist_em(symbol="小金属", start_date="20211201", end_date="20250212", period="日k", adjust="")

同花顺  行业板块

In [ ]:
ak.stock_board_industry_summary_ths()

In [ ]:
ak.stock_board_industry_index_ths(symbol="元件", start_date="20240101", end_date="20250212")

股票热度 关注 讨论 交易

关注排行榜 symbol="最热门"; choice of {"本周新增", "最热门"}


In [ ]:
ak.stock_hot_follow_xq(symbol="最热门")

In [ ]:
ak.stock_hot_tweet_xq(symbol="最热门")

In [ ]:
ak.stock_hot_deal_xq(symbol="最热门")

问财

In [ ]:
ak.stock_hot_rank_wc(date="20250212")

东财 单次返回当前交易日前 100 个股票的人气排名数据

In [ ]:
ak.stock_hot_rank_em()

飙升榜-A股

In [ ]:
ak.stock_hot_up_em()

历史趋势及粉丝特征

In [ ]:
ak.stock_hot_rank_detail_em(symbol="SH600996")

热门关键词

In [ ]:
ak.stock_hot_keyword_em(symbol="SH600996")

内部交易

In [ ]:
ak.stock_inner_trade_xq()

热搜股票 time="今日"; choice of {"今日", "1小时"}

In [ ]:
ak.stock_hot_search_baidu(symbol="A股", date="20240213", time="今日")

强势股池 

In [ ]:
ak.stock_zt_pool_strong_em(date='20250206')

In [ ]:
ak.stock_zt_pool_sub_new_em(date='20250123')

In [ ]:
ak.stock_info_cjzc_em()

In [ ]:
ak.stock_info_global_em()

In [ ]:
ak.stock_info_global_sina()

In [ ]:
ak.stock_info_global_futu()

In [ ]:
ak.stock_info_global_ths()

In [ ]:
ak.stock_info_global_cls(symbol="全部")

技术指标 symbol="创月新高"; choice of {"创月新高", "半年新高", "一年新高", "历史新高"}

创新高cxg 创新低cxd 

连续上涨lxsz 连续下跌lxxd 持续放量cxfl 持续缩量cxsl 向上突破xstp 向下突破xxtp 量价齐升ljqs 量价齐跌ljqd 险资举牌xzjp

In [ ]:
ak.stock_rank_cxg_ths(symbol="创月新高")

In [ ]:
ak.stock_rank_cxd_ths(symbol="创月新低")

In [ ]:
ak.stock_rank_lxsz_ths()

In [ ]:
ak.stock_rank_cxfl_ths()

In [ ]:
ak.stock_rank_cxsl_ths()

In [ ]:
ak.stock_rank_xzjp_ths()

ESG评级

In [241]:
ak.stock_esg_rate_sina()

,成分股代码,评级机构,评级,评级季度,标识,交易市场
0,SZ000001,国证指数,AA,2024Q2,NaN,CN
1,SZ000001,中国国新,BBB,2024Q4,NaN,CN
2,SZ000001,中财绿金院,A-,2023Q4,NaN,CN
3,SZ000001,商道融绿,A-,2024Q4,NaN,CN
4,SZ000001,盟浪,A,2022Q4,NaN,CN
...,...,...,...,...,...,...
33295,SZ300539,盟浪,B,2022Q4,NaN,CN
33296,SZ300539,中诚信,BB,2025Q1,NaN,CN
33297,SZ300539,晨星,-,2023Q4,NaN,CN
33298,SZ300539,妙盈,-,2024Q1,NaN,CN


In [240]:
ak.stock_esg_msci_sina()

,股票代码,ESG评分,环境总评,社会责任总评,治理总评,评级日期,交易市场
0,000513.SZ,AAA,7.0,6.4,6.3,2024-07-29,CN
1,00066.HK,AAA,6.7,5.8,6.8,2025-01-22,HK
2,00939.HK,AAA,7.4,7.7,6.9,2024-12-17,HK
3,00992.HK,AAA,5.2,7.0,5.5,2024-12-17,HK
4,01310.HK,AAA,10.0,8.0,7.7,2024-07-03,HK
...,...,...,...,...,...,...,...
4738,VTS.US,CCC,1.1,2.4,5.8,2024-10-07,US
4739,WLFC.US,CCC,2.6,4.8,3.9,2025-01-29,US
4740,WULF.US,CCC,0.3,0.9,4.6,2024-11-20,US
4741,XYIGY.US,CCC,4.1,4.2,1.7,2024-12-04,US


In [239]:
ak.stock_esg_rft_sina()

,股票代码,ESG评分,ESG评分日期,环境总评,环境总评日期,社会责任总评,社会责任总评日期,治理总评,治理总评日期,争议总评,争议总评日期,行业,交易所
0,SBS,52.2(B-),2025-01-11,44.2(C+),2025-01-11,62.9(B),2025-01-11,52.9(B-),2025-01-11,100.0(A+),2024-05-04,公共用水,纽交所
1,600893.SH,32.3(C-),2025-01-11,33.9(C),2025-01-04,18.9(D+),2025-01-04,49.0(C+),2025-01-11,100.0(A+),2024-05-04,航空装备Ⅱ,上交所
2,LNG,74.3(B+),2025-01-11,83.3(A-),2025-01-11,64.0(B),2025-01-11,78.6(A-),2025-01-11,74.3(B+),2025-01-11,油气管道,纽交所
3,002643.SZ,68.9(B+),2025-01-11,64.5(B),2025-01-11,79.4(A-),2025-01-11,60.5(B),2025-01-11,100.0(A+),2024-11-23,电子化学品Ⅱ,深交所
4,01877.HK,46.5(C+),2025-01-11,52.2(B-),2025-01-11,44.8(C+),2025-01-11,42.4(C+),2025-01-11,100.0(A+),2024-04-20,药品及生物科技,港交所
...,...,...,...,...,...,...,...,...,...,...,...,...,...
95,WWW,38.2(C),2025-01-11,56.7(B-),2025-01-11,38.6(C),2025-01-11,25.6(C-),2025-01-11,76.0(A-),2024-11-09,服装-鞋类,纽交所
96,FENG,16.6(D),2025-01-11,0.0(D-),2024-05-04,10.5(D),2025-01-11,30.0(C-),2025-01-11,100.0(A+),2024-05-04,互联网软件-服务,纽交所
97,ATOS,27.7(C-),2025-01-11,0.0(D-),2024-05-04,44.5(C+),2025-01-11,30.9(C-),2025-01-11,100.0(A+),2024-05-04,其他制药,纳斯达克
98,GDDY,54.5(B-),2025-01-11,34.8(C),2025-01-11,66.3(B),2025-01-11,50.3(B-),2025-01-11,80.1(A-),2025-01-11,互联网软件-服务,纽交所


In [238]:
ak.stock_esg_zd_sina()

,股票代码,ESG评分,环境总评,社会责任总评,治理总评,评分日期
0,600276.SH,90.02(AAA),98.13(AAA),78.99(AA),91.85(AAA),2025-01-31
1,300274.SZ,88.91(AAA),96.31(AAA),74.9(AA),93.64(AAA),2025-01-31
2,600406.SH,87.37(AAA),91.97(AAA),78.23(AA),90.73(AAA),2025-01-31
3,688063.SH,86.26(AAA),98.96(AAA),65.68(A),91.08(AAA),2025-01-31
4,688271.SH,85.98(AAA),99.03(AAA),72.64(AA),89.35(AAA),2025-01-31
...,...,...,...,...,...,...
95,603687.SH,81.98(AAA),92.93(AAA),67.84(A),80.75(AAA),2025-01-31
96,000727.SZ,81.95(AAA),99.42(AAA),62.92(A),81.18(AAA),2025-01-31
97,603063.SH,81.95(AAA),95.71(AAA),62.14(A),84.78(AAA),2025-01-31
98,002643.SZ,81.94(AAA),89.6(AAA),68.98(A),83.75(AAA),2025-01-31


In [237]:
ak.stock_esg_hz_sina()

,日期,股票代码,交易市场,股票名称,ESG评分,ESG等级,环境,环境等级,社会,社会等级,公司治理,公司治理等级
0,NaT,300760.SZ,cn,迈瑞医疗,100.00,AAA,78.38,BB,94.08,AA,93.94,AA
1,NaT,300896.SZ,cn,爱美客,99.88,AAA,87.46,A,91.98,AA,90.70,AA
2,NaT,601825.SH,cn,沪农商行,98.59,AAA,86.23,A,89.60,A,90.52,AA
3,NaT,600549.SH,cn,厦门钨业,98.00,AAA,90.37,AA,90.73,AA,85.62,A
4,NaT,688676.SH,cn,金盘科技,97.84,AAA,89.07,A,90.93,AA,87.11,A
...,...,...,...,...,...,...,...,...,...,...,...,...
6099,NaT,000576.SZ,cn,甘化科工,50.00,C,58.70,C,74.13,B,74.56,B
6100,NaT,000571.SZ,cn,新大洲A,50.00,C,61.02,CC,60.48,CC,64.65,CC
6101,NaT,000564.SZ,cn,供销大集,50.00,C,66.15,CCC,72.92,B,68.73,CCC
6102,NaT,000421.SZ,cn,南京公用,50.00,C,60.78,CC,75.61,BB,67.93,CCC


A股股票指数数据 symbol="上证系列指数"；choice of {"沪深重要指数", "上证系列指数", "深证系列指数", "指数成份", "中证系列指数"}


In [246]:
ak.stock_zh_index_spot_em(symbol="中证系列指数")

,序号,代码,名称,最新价,涨跌幅,涨跌额,成交量,成交额,振幅,最高,最低,今开,昨收,量比
0,1,930781,中证影视,1865.28,5.82,102.66,40134799,4.150761e+10,6.35,1888.09,1776.09,1792.29,1762.62,1.59
1,2,CESG10,中华博彩指数,2485.45,5.13,121.20,1127591,1.607782e+09,5.04,2505.67,2386.61,2394.19,2364.25,1.35
2,3,H11049,AMAC文体,1382.25,4.11,54.63,38629482,3.757918e+10,5.36,1401.02,1329.82,1338.70,1327.62,1.35
3,4,930622,CSSW白酒,40449.54,3.46,1353.49,3197117,2.570055e+10,4.48,40812.76,39060.98,39108.13,39096.05,2.54
4,5,931481,SHS线上消费,1041.67,3.10,31.30,68090043,1.479994e+11,4.62,1065.68,1018.98,1022.51,1010.37,1.42
...,...,...,...,...,...,...,...,...,...,...,...,...,...,...
500,501,931079,5G通信,1019.20,-2.84,-29.84,29552612,8.831845e+10,2.62,1044.33,1016.82,1044.33,1049.04,0.89
501,502,931144,通信技术,1135.34,-2.91,-34.02,20738542,7.810564e+10,2.69,1164.03,1132.61,1164.03,1169.36,1.02
502,503,932087,集成电路,839.54,-2.99,-25.84,14015464,8.840649e+10,3.05,864.57,838.15,864.57,865.38,0.92
503,504,931136,深圳科技,7752.35,-3.10,-248.27,12620776,4.735960e+10,2.78,7956.40,7733.94,7956.40,8000.62,0.88


历史行情数据-东方财富 period="daily"; choice of {'daily', 'weekly', 'monthly'}

In [253]:
ak.index_zh_a_hist(symbol="931144", period="daily", start_date="20250212", end_date="20250212")

,日期,开盘,收盘,最高,最低,成交量,成交额,振幅,涨跌幅,涨跌额,换手率
0,2025-02-12,1141.51,1169.36,1169.54,1141.05,21223921,7.683352e+10,2.49,2.28,26.04,4.43


中国股票指数成份 

In [254]:
ak.index_stock_info()

,index_code,display_name,publish_date
0,000001,上证指数,1991-07-15
1,000002,A股指数,1992-02-21
2,000003,B股指数,1992-02-21
3,000004,工业指数,1993-05-03
4,000005,商业指数,1993-05-03
...,...,...,...
720,399994,中证信息安全主题指数,2015-03-12
721,399995,中证基建工程指数,2015-03-12
722,399996,中证智能家居指数,2014-09-17
723,399997,中证白酒指数,2015-01-21


In [255]:
ak.index_stock_cons(symbol="000300")

,品种代码,品种名称,纳入日期
0,600482,中国动力,2024-12-16
1,688472,阿特斯,2024-12-16
2,600377,宁沪高速,2024-12-16
3,002028,思源电气,2024-12-16
4,600160,巨化股份,2024-12-16
...,...,...,...
295,000425,徐工科技,2005-04-08
296,000157,中联重科,2005-04-08
297,000063,中兴通讯,2005-04-08
298,000001,深发展A,2005-04-08


In [256]:
ak.index_stock_cons_csindex(symbol="000300")

,日期,指数代码,指数名称,指数英文名称,成分券代码,成分券名称,成分券英文名称,交易所,交易所英文名称
0,2025-02-12,000300,沪深300,CSI 300,000001,平安银行,"Ping An Bank Co., Ltd.",深圳证券交易所,Shenzhen Stock Exchange
1,2025-02-12,000300,沪深300,CSI 300,000002,万科A,China Vanke Co Ltd,深圳证券交易所,Shenzhen Stock Exchange
2,2025-02-12,000300,沪深300,CSI 300,000063,中兴通讯,ZTE Corporation,深圳证券交易所,Shenzhen Stock Exchange
3,2025-02-12,000300,沪深300,CSI 300,000100,TCL科技,TCL Technology Group Corporation,深圳证券交易所,Shenzhen Stock Exchange
4,2025-02-12,000300,沪深300,CSI 300,000157,中联重科,Zoomlion Heavy Industry Science & Technology C...,深圳证券交易所,Shenzhen Stock Exchange
...,...,...,...,...,...,...,...,...,...
295,2025-02-12,000300,沪深300,CSI 300,688396,华润微,China Resources Microelectronics Limited,上海证券交易所,Shanghai Stock Exchange
296,2025-02-12,000300,沪深300,CSI 300,688472,阿特斯,"CSI Solar Co., Ltd.",上海证券交易所,Shanghai Stock Exchange
297,2025-02-12,000300,沪深300,CSI 300,688506,百利天恒,"Sichuan Biokin Pharmaceutical Co.,Ltd.",上海证券交易所,Shanghai Stock Exchange
298,2025-02-12,000300,沪深300,CSI 300,688599,天合光能,"Trina Solar Co., Ltd.",上海证券交易所,Shanghai Stock Exchange


In [257]:
ak.sw_index_first_info()

,行业代码,行业名称,成份个数,静态市盈率,TTM(滚动)市盈率,市净率,静态股息率
0,801010.SI,农林牧渔,101,24.60,20.77,2.22,1.59
1,801030.SI,基础化工,397,20.93,21.77,1.96,2.14
2,801040.SI,钢铁,45,12.53,15.50,0.98,3.69
3,801050.SI,有色金属,136,20.89,17.79,2.28,1.97
4,801080.SI,电子,458,56.12,49.26,3.99,0.90
5,801880.SI,汽车,268,26.49,25.71,2.43,1.52
6,801110.SI,家用电器,95,16.46,15.66,2.58,3.40
7,801120.SI,食品饮料,123,21.20,19.74,4.70,3.50
8,801130.SI,纺织服饰,104,17.93,18.48,1.76,4.02
9,801140.SI,轻工制造,154,20.58,19.98,2.04,2.77


In [258]:
ak.sw_index_second_info()

,行业代码,行业名称,上级行业,成份个数,静态市盈率,TTM(滚动)市盈率,市净率,静态股息率
0,801016.SI,种植业,农林牧渔,19,34.25,36.18,2.58,1.58
1,801015.SI,渔业,农林牧渔,6,28.03,29.11,0.99,1.09
2,801014.SI,饲料,农林牧渔,16,38.05,26.40,4.40,0.91
3,801012.SI,农产品加工,农林牧渔,22,33.81,43.56,1.91,1.43
4,801017.SI,养殖业,农林牧渔,21,9.57,5.94,1.90,1.96
...,...,...,...,...,...,...,...,...
119,801963.SI,炼化及贸易,石油石化,31,10.98,11.49,1.03,5.06
120,801971.SI,环境治理,环保,106,17.81,16.82,1.40,1.96
121,801972.SI,环保设备Ⅱ,环保,28,23.16,24.07,1.58,2.93
122,801981.SI,个护用品,美容护理,13,32.95,31.99,2.49,2.05


In [259]:
ak.sw_index_third_info()

,行业代码,行业名称,上级行业,成份个数,静态市盈率,TTM(滚动)市盈率,市净率,静态股息率
0,850111.SI,种子,种植业,8,48.23,52.43,3.57,0.65
1,850113.SI,其他种植业,种植业,5,67.89,64.00,2.19,0.35
2,850122.SI,水产养殖,渔业,4,66.78,78.42,0.97,0.45
3,850142.SI,畜禽饲料,饲料,9,27.35,61.73,1.76,1.41
4,850151.SI,果蔬加工,农产品加工,5,19.71,31.83,3.04,2.70
...,...,...,...,...,...,...,...,...
253,859714.SI,综合环境治理,环境治理,15,10.60,10.05,1.86,1.85
254,859721.SI,环保设备Ⅲ,环保设备Ⅱ,28,23.16,24.07,1.58,2.93
255,859811.SI,生活用纸,个护用品,9,28.16,29.55,2.17,2.34
256,859821.SI,化妆品制造及其他,化妆品,7,23.10,27.50,2.10,2.79


申万三级行业成份 symbol="850111.SI"; 行业代码; 可以通过 ak.sw_index_third_info() 获取所有行业代码


In [260]:
ak.sw_index_third_cons(symbol="859822.SI")

,序号,股票代码,股票简称,纳入时间,申万1级,申万2级,申万3级,价格,市盈率,市盈率ttm,市净率,股息率,市值,归母净利润同比增长(09-30),归母净利润同比增长(06-30),营业收入同比增长(09-30),营业收入同比增长(06-30)
0,1,300957.SZ,贝泰妮,2021-04-01,—,—,品牌化妆品,41.66,23.32,29.79,2.99,1.43,176.47,-28.39,7.50,17.09,18.45
1,2,300740.SZ,水羊股份,2018-02-22,—,—,品牌化妆品,12.02,15.87,22.39,2.34,0.83,46.68,-47.60,-25.74,-9.84,0.14
2,3,600315.SH,上海家化,2001-03-22,—,—,品牌化妆品,16.04,21.56,40.14,1.41,1.63,107.83,-58.72,-20.93,-12.07,-8.51
3,4,603605.SH,珀莱雅,2017-11-22,—,—,品牌化妆品,83.80,27.81,22.95,6.97,1.08,332.06,33.95,40.48,32.72,37.90
4,5,603630.SH,拉芳家化,2017-03-20,—,—,品牌化妆品,12.55,43.15,63.92,1.46,2.21,28.26,-28.03,-31.64,8.65,15.17
5,6,600223.SH,福瑞达,2024-02-06,—,—,品牌化妆品,7.21,24.16,30.99,1.82,2.08,73.29,-28.09,-33.18,-17.45,-22.15
6,7,301371.SZ,敷尔佳,2023-08-08,—,—,品牌化妆品,34.75,18.55,19.13,2.52,4.32,139.03,-4.20,-3.71,9.47,8.17
7,8,603983.SH,丸美生物,2019-08-01,—,—,品牌化妆品,31.36,48.48,38.77,3.84,2.46,125.75,37.38,35.09,27.07,27.65


In [261]:
ak.drewry_wci_index(symbol="composite")

,date,wci
0,2016-03-10,700.57
1,2016-03-17,674.41
2,2016-03-24,666.27
3,2016-03-31,849.08
4,2016-04-07,868.06
...,...,...
456,2025-01-09,3986.00
457,2025-01-16,3855.00
458,2025-01-23,3455.00
459,2025-01-30,3364.00


In [262]:
ak.index_neaw_cx()

,日期,新经济行业入职平均工资水平,变化值
0,2015-08-01,7409.0000,0.0000
1,2015-09-01,7307.0000,-102.0000
2,2015-10-01,7367.0000,60.0000
3,2015-11-01,7298.0000,-69.0000
4,2015-12-01,7494.0000,196.0000
...,...,...,...
109,2024-09-01,12897.9464,-21.2033
110,2024-10-01,12548.0579,-349.8885
111,2024-11-01,12738.9296,190.8717
112,2024-12-01,12590.0503,-148.8793


汽车销量排行

In [282]:
ak.car_market_cate_cpca(symbol="占比", indicator="零售")

,月份,MPV,SUV,轿车
0,2023-12月,4.228617,49.161808,46.609570
1,2024-1月,4.518923,50.674800,44.806282
2,2024-2月,4.744053,49.406860,45.849090
3,2024-3月,5.498287,47.672585,46.829132
4,2024-4月,5.264448,47.435260,47.300285
5,2024-5月,4.533718,48.472427,46.993862
6,2024-6月,4.791434,49.689570,45.518986
7,2024-7月,4.868580,49.128750,46.002666
8,2024-8月,4.814876,48.952140,46.232980
9,2024-9月,4.614374,49.194850,46.190780


In [283]:
ak.car_market_cate_cpca(symbol="轿车", indicator="零售")

,月份,2024年,2023年
0,1月,91.3324,60.9505
1,2月,50.7427,65.6146
2,3月,79.1727,76.0359
3,4月,72.5401,77.9511
4,5月,80.4439,85.1469
5,6月,80.1721,89.6155
6,7月,78.9190,83.7117
7,8月,88.1810,89.3844
8,9月,97.4152,92.8885
9,10月,103.2780,93.3454


盖世研究院 symbol="车型榜"; choice of {"车企榜", "品牌榜", "车型榜"}

In [284]:
ak.car_sale_rank_gasgoo(symbol="车企榜", date="202412")

,厂商,2024-12,12月同比,12月环比,2024-1到12,2023-1到12,2022-1到12
0,比亚迪汽车,509440,49.76%,-88.01%,4250370,3012906.0,1862603.0
1,奇瑞汽车,283903,44.51%,-88.49%,2466218,1715602.0,1112193.0
2,吉利汽车,217450,46.72%,-90.31%,2243836,1664161.0,1394986.0
3,一汽大众,170904,-13.91%,-89.32%,1600859,1847730.0,1801645.0
4,长安汽车,149883,31.26%,-90.88%,1642634,1592443.0,1380432.0
5,上汽通用五菱,134144,-12.23%,-85.18%,905034,835438.0,1098380.0
6,上汽大众,129990,-8.87%,-88.68%,1148091,1215003.0,1320833.0
7,长城汽车,118852,23.25%,-88.74%,1055235,1027847.0,880808.0
8,一汽丰田,100570,12.13%,-87.07%,778103,800099.0,834617.0
9,特斯拉汽车,93766,-0.4%,-89.77%,916660,947742.0,710865.0


In [285]:
ak.car_sale_rank_gasgoo(symbol="品牌榜", date="202412")

,品牌,2024-12,12月同比,12月环比,2024-1到12,2023-1到12,2022-1到12
0,比亚迪,482652,50.09%,-88.11%,4060493,2877353.0,1852625.0
1,大众,219817,-9.02%,-89.11%,2018653,2221716.0,2333740.0
2,丰田,177585,-4.82%,-88.25%,1510819,1745950.0,1833941.0
3,奇瑞,174430,26.39%,-89.28%,1626886,1272568.0,878690.0
4,本田,125491,-6.61%,-85.94%,892773,1228186.0,1389885.0
5,吉利,105077,35.06%,-91.53%,1240119,1052053.0,1013870.0
6,长安,99370,62.67%,-91.61%,1183836,1155825.0,1126602.0
7,特斯拉,93766,-0.4%,-89.77%,916660,947742.0,710865.0
8,哈弗,83425,27.21%,-88.19%,706234,715188.0,616550.0
9,捷途,66020,56.97%,-88.38%,568387,315167.0,180067.0


In [286]:
ak.car_sale_rank_gasgoo(symbol="车型榜", date="202412")

,车型,2024-12,12月同比,12月环比,2024-1到12,2023-1到12,2022-1到12
0,Model Y,61984,-0.28%,-88.87%,556689,646845.0,455091.0
1,海鸥,57087,12.99%,-88.09%,479294,280217.0,NaN
2,宏光MINI EV,44903,-11.19%,-80.08%,225371,118834.0,554067.0
3,宋PLUS DM,43139,17.79%,-88.87%,387572,333249.0,390953.0
4,秦L DM-i,42700,4270000%,-84.98%,284345,NaN,NaN
5,瑞虎8,35856,38.92%,-88.6%,314397,225586.0,180450.0
6,元PLUS,32930,-19.71%,-89.36%,309536,412202.0,202058.0
7,帕萨特,32204,37.87%,-87.4%,255575,201809.0,174440.0
8,宋Pro DM,31790,36.37%,-88.68%,280755,203491.0,20073.0
9,Model 3,31782,-0.62%,-91.17%,359971,300897.0,255774.0


电影票房

In [309]:
ak.movie_boxoffice_realtime()

,排序,影片名称,实时票房,票房占比,上映天数,累计票房
0,1,哪吒之魔童闹海,22588.22,86.47,16,972652.62
1,2,唐探1900,1936.31,7.41,16,299091.44
2,3,熊出没·重启未来,684.92,2.62,16,69305.62
3,4,封神第二部：战火西岐,336.92,1.29,16,113815.11
4,5,蛟龙行动,259.52,0.99,16,37393.44
5,6,射雕英雄传：侠之大者,173.88,0.67,16,63622.40
6,7,美国队长4,114.39,0.44,0,114.39
7,8,雄狮少年2,11.64,0.04,62,8131.59
8,9,误杀3,4.12,0.02,48,93254.76
9,10,名侦探柯南：迷宫的十字路口,1.80,0.01,49,14764.24


In [317]:
ak.movie_boxoffice_daily(date="20250201")

,排序,影片名称,单日票房,环比变化,累计票房,平均票价,场均人次,口碑指数,上映天数
0,1,哪吒之魔童闹海,61969,18.0,232056,51,4,9.35,77
1,2,唐探1900,36328,-12.0,154311,51,4,8.72,49
2,3,封神第二部：战火西岐,13830,-35.0,82785,51,4,8.38,28
3,4,熊出没·重启未来,7672,-19.0,36460,48,4,9.03,30
4,5,射雕英雄传：侠之大者,8578,-54.0,51589,47,4,8.97,27
5,6,蛟龙行动,3721,-16.0,21056,52,4,8.51,21
6,7,误杀3,10,-10.0,93128,43,36,8.38,8
7,8,地球脉动：极境生存,8,0.0,461,30,190,NaN,121
8,9,异乡来客,0,NaN,46,42,44,0.00,106
9,10,雄狮少年2,3,0.0,8053,45,50,9.04,22


In [314]:
ak.movie_boxoffice_yearly(date="2025")

,排序,影片名称,类型,总票房,平均票价,场均人次,国家及地区,上映日期
0,1,哪吒之魔童闹海,动画,969421,49,59.0,中国,2025-01-29
1,2,唐探1900,喜剧,298730,50,37.0,中国,2025-01-29
2,3,封神第二部：战火西岐,动作,113752,52,29.0,中国,2025-01-29
3,4,熊出没·重启未来,动画,69205,48,24.0,中国,2025-01-29
4,5,射雕英雄传：侠之大者,动作,63610,49,33.0,中国,2025-01-29
5,6,误杀3,剧情,63354,42,6.0,中国,2024-12-28
6,7,小小的我,剧情,39439,40,5.0,中国,2024-12-27
7,8,蛟龙行动,剧情,37352,50,18.0,中国,2025-01-29
8,9,“骗骗”喜欢你,爱情,34306,35,5.0,中国,2024-12-31
9,10,名侦探柯南：迷宫的十字路口,动画,7587,38,4.0,日本,2024-12-27


In [301]:
ak.video_tv()

,排序,名称,类型,播映指数,媒体热度,用户热度,好评度,观看度,统计日期
0,1,五福临门,喜剧/爱情,83.78,88.58,83.24,59.16,92.83,2025-02-12
1,2,六姊妹,剧情,81.39,83.58,79.13,57.31,93.02,2025-02-12
2,3,无所畏惧之永不放弃,都市,80.84,81.08,79.75,54.67,93.03,2025-02-12
3,4,仙台有树,古装/爱情,79.36,79.82,76.94,56.72,91.29,2025-02-12
4,5,掌心,古装/悬疑,78.74,79.57,72.50,59.15,93.02,2025-02-12
5,6,三叉戟2,剧情,78.70,78.51,73.57,70.70,87.34,2025-02-12
6,7,白色橄榄树,剧情/爱情,78.05,80.28,75.85,56.20,88.65,2025-02-12
7,8,余烬之上,剧情/悬疑,77.86,85.23,73.49,52.77,89.83,2025-02-12
8,9,白月梵星,剧情/爱情,76.54,70.73,75.95,56.86,88.06,2025-02-12
9,10,大奉打更人,喜剧/奇幻,76.50,75.56,75.05,54.99,87.58,2025-02-12


In [302]:
ak.video_variety_show()

,排序,名称,类型,播映指数,媒体热度,用户热度,好评度,观看度,统计日期
0,1,一路繁花,真人秀,77.01,82.18,82.65,54.90,78.63,2025-02-12
1,2,大侦探拾光季,真人秀,76.36,71.15,74.00,54.37,90.39,2025-02-12
2,3,火星情报局第七季,真人秀/脱口秀,75.71,46.23,90.64,52.26,83.47,2025-02-12
3,4,太阳市集,真人秀,72.09,77.02,67.30,56.03,81.64,2025-02-12
4,5,快乐再出发山海季,真人秀,70.79,39.32,68.69,83.64,80.86,2025-02-12
5,6,声生不息·大湾区季,真人秀,69.92,66.48,59.45,68.99,82.28,2025-02-12
6,7,团建不能停,真人秀,67.95,76.51,53.25,65.27,80.12,2025-02-12
7,8,2025年中央广播电视总台元宵晚会,晚会,65.25,69.63,48.28,56.31,84.18,2025-02-12
8,9,我家那小子·好好生活季,真人秀,63.34,47.31,61.01,56.20,75.62,2025-02-12
9,10,奔跑吧茶马古道篇,真人秀,63.01,55.27,49.13,59.86,81.55,2025-02-12


In [303]:
ak.business_value_artist()

,排名,艺人,商业价值,专业热度,关注热度,预测热度,美誉度,统计日期
0,1,杨紫,88.07,85.06,80.94,90.24,65.67,2025-02-13
1,2,檀健次,85.23,82.80,80.61,65.48,64.00,2025-02-13
2,3,杨幂,85.18,84.65,76.82,68.91,53.33,2025-02-13
3,4,王一博,84.44,80.89,80.17,76.48,65.33,2025-02-13
4,5,白鹿,83.99,77.67,82.40,91.08,61.33,2025-02-13
...,...,...,...,...,...,...,...,...
95,96,丁程鑫,72.25,62.80,77.16,82.41,62.00,2025-02-13
96,97,王珞丹,72.22,73.10,65.18,42.19,64.67,2025-02-13
97,98,张远,72.19,71.18,68.91,38.44,82.33,2025-02-13
98,99,尹正,72.14,73.15,61.81,63.68,61.33,2025-02-13


In [304]:
ak.online_value_artist()

,排名,艺人,流量价值,专业热度,关注热度,预测热度,带货力,统计日期
0,1,肖战,89.40,68.05,85.05,85.77,96.29,2025-02-13
1,2,白鹿,88.81,77.67,82.40,91.08,91.40,2025-02-13
2,3,杨紫,88.56,85.06,80.94,90.24,91.61,2025-02-13
3,4,迪丽热巴,88.39,75.81,79.60,97.80,94.06,2025-02-13
4,5,成毅,87.32,71.00,82.75,77.88,97.80,2025-02-13
...,...,...,...,...,...,...,...,...
95,96,沈月,73.18,58.30,73.98,53.97,75.89,2025-02-13
96,97,秦岚,73.13,72.10,65.13,82.25,73.49,2025-02-13
97,98,梅婷,73.12,55.93,76.19,73.23,48.96,2025-02-13
98,99,胡先煦,73.08,68.13,66.11,76.62,77.51,2025-02-13


微博舆情报告 time_period="CNHOUR12"

CNHOUR2	2小时
CNHOUR6	6小时
CNHOUR12	12小时
CNHOUR24	1天
CNDAY7	1周
CNDAY30	1月

In [307]:
ak.stock_js_weibo_report(time_period="CNDAY30")

,name,rate
0,比亚迪,-0.75
1,润和软件,-4.26
2,华银电力,-0.18
3,士兰微,0.14
4,东方财富,1.75
5,长安汽车,-1.01
6,朗姿股份,-3.21
7,贵州茅台,3.01
8,酒鬼酒,1.76
9,隆基股份,-0.39
